FabNESO/tasks.py

```diff
@@ -309,18 +314,22 @@ def neso_vbmc(
 
     Args:
         config: Directory with ensemble configuration information.
+        reference_field_file: Name of a numpy file that holds a reference field
+            measurement for the calibration run.
```

Comment:
```suggestion
        reference_field_file: Name of a NumPy .txt file that holds a reference
            field measurement for the calibration run.
```
Making the expected file type explicit probably helpful as NumPy can output to a few different formats (`.npy` / `.npz` / `.txt`).